static/resources/yaml/datadog-agent-npm.yaml

```diff
@@ -498,12 +525,16 @@ spec:
         - hostPath:
             path: /var/run
           name: runtimesocketdir
+        - name: tmpdir
+          emptyDir: {}
         - hostPath:
             path: /proc
           name: procdir
         - hostPath:
             path: /sys/fs/cgroup
           name: cgroups
+        - name: s6-run
+          emptyDir: {}
```

Comment:
This looks like a bug in the Helm chart rather than in this PR, but a new volume is declared here whereas it is mounted in no container.
So, it’s useless.
And it’s expected to be useless since there’s no S6 in the multi-container setup.